test/session.cpp

```diff
@@ -0,0 +1,617 @@
+/******************************************************************************
+ *                                                                            *
+ * Copyright (C) 2025 MachineWare GmbH                                        *
+ * All Rights Reserved                                                        *
+ *                                                                            *
+ * This is work is licensed under the terms described in the LICENSE file     *
+ * found in the root directory of this source tree.                           *
+ *                                                                            *
+ ******************************************************************************/
+
+#include "testing.h"
+
+using namespace vsp;
+
+using testing::AllOf;
+using testing::ContainsRegex;
+using testing::Each;
+using testing::ElementsAre;
+using testing::SizeIs;
+
+class session_test : public testing::Test
+{
+protected:
+    session_test(): sess("localhost", 4444) {
+        subp.run("system");
```

Comment:
I dont think system was a good name choice... can we rename this to simplevp? I am worried about windows defender... also there is a dangerous syscall with that name...